python/gui/MaaS/cbv/MapView.py

```diff
@@ -142,3 +192,56 @@ def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
 
         # Return the rendered page
         return render(request, 'maas/map.html', payload)
+
+
+class DomainView(View):
+
+    """
+    A view used to render the map
+    """
+    def get(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
+        """
+        The handler for 'get' requests.  This will render the 'map.html' template
+
+        :param HttpRequest request: The request asking to render this page
+        :param args: An ordered list of arguments
+        :param kwargs: A dictionary of named arguments
+        :return: A rendered page
+        """
+        # If a list of error messages wasn't passed, create one
+        if 'errors' not in kwargs:
```

Comment:
A list of errors and stuff wouldn't get passed in here - it would be path variables. If the path to the domain view was something along the lines of `/domain/(?P<one>...)/(?P<two>...)/(?P<three>...)`, I believe the key value pairs for `one`, `two`, and `three` would end up in kwargs. Things like list of errors, warnings, and info would come in through `request.GET`.